src/cratedb_about/outline/cratedb-outline.yaml

```diff
@@ -293,6 +293,8 @@ data:
         They also influence the behaviour when the records are queried.
       parents: [ sql-syntax ]
       tags: [ sql ]
+      # FIXME: This needs about 40_000 input tokens. Maybe a stripped-down variant could help?
+      enabled: false
```

Comment:
The section `# SQL: Data types` needed to be disabled to save a whole bunch of input tokens.

> FIXME: This needs about 40_000 input tokens. Maybe a stripped-down variant could help?

Please bring it back.